hikari/api/special_endpoints.py

```diff
@@ -1539,6 +1641,234 @@ def add_to_container(self) -> _ContainerT:
         """
 
 
+class TextInputBuilder(ComponentBuilder, abc.ABC, typing.Generic[_ContainerT]):
```

Comment:
```suggestion
class ModalTextInputBuilder(ComponentBuilder, abc.ABC, typing.Generic[_ContainerT]):
```

Reply:
Unfortunately, I have decided to decline this suggestion.